plugin/pom.xml

```diff
@@ -21,7 +21,7 @@
   <url>https://github.com/jenkinsci/warnings-ng-plugin</url>
 
   <properties>
-    <revision>8.4.0</revision>
+    <revision>9.0.0</revision>
```

Comment:
Is this version number ok? Some incompatible changes (new property for the pipeline script) added.

Reply:
don't worry about the version it will be set during release

```suggestion
    <revision>8.4.0</revision>
```

Reply:
but it will always fail CI which is annoying for development before the release

Reply:
why does it fail CI?

Reply:
since it breaks the semantic versioning:

```
[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.11.4:check (run-revapi) on project warnings-ng: The following API problems caused the build to fail:
[ERROR] java.field.serialVersionUIDChanged: field io.jenkins.plugins.analysis.core.steps.PublishIssuesStep.serialVersionUID: serialVersionUID constant changed. Objects serialized using the old version of the class will no longer be deserializable. (breaks semantic versioning)
[ERROR] java.field.serialVersionUIDChanged: field io.jenkins.plugins.analysis.core.steps.RecordIssuesStep.serialVersionUID: serialVersionUID constant changed. Objects serialized using the old version of the class will no longer be deserializable. (breaks semantic versioning)
```

Reply:
Ah ok